packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -341,11 +340,6 @@ export class TeamsAppSolution implements Solution {
     const solutionSettings = settingsRes.value;
     const selectedPlugins = await this.reloadPlugins(solutionSettings);
 
-    const scaffoldLocalDebugSettingResult = await scaffoldLocalDebugSettingsV1(ctx);
-    if (scaffoldLocalDebugSettingResult.isErr()) {
-      return scaffoldLocalDebugSettingResult;
-    }
-
     const results: Result<any, FxError>[] = await Promise.all<Result<any, FxError>>(
```

Comment:
When to remove the whole `migrate` method?

Reply:
In the next PR, it contains removing the migrate v1 logic in core & solution.